test_scripts/Defects/5_0/1846_Mismatch_for_mandatory_parameter_between_MOBILE_API_and_HMI_API.lua

```diff
@@ -0,0 +1,71 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1846 
+--
+-- Description:
+-- Mismatch for mandatory parameter between MOBILE_API and HMI_API
+--
+-- There is a mismatch for the mandatory parameter between the MOBILE_API and the HMI_API for the 
+-- following parameters:
+--     distanceToManeuver
+--     distanceToManeuverScale
+-- The MOBILE_API is reflecting that these parameters are not mandatory, whereas the HMI_API 
+-- reflects that they are mandatory.
+--
+-- Preconditions:
+-- 1) Clear environment
+-- 2) SDL started, HMI and mobile session connected
+-- 3) Registered and activated app
+-- 5) PTU
+--
+-- Steps:
+-- 1) send mobile RPC "ShowConstantTBT" without "distanceToManeuver" param
+--    and recieve resultCode = "INVALID_DATA"
+-- 2) send mobile RPC "ShowConstantTBT" without "distanceToManeuverScale" param
+--    and recieve resultCode = "INVALID_DATA"
+-- 3) send mobile RPC "ShowConstantTBT" with "distanceToManeuver" and 
+--    "distanceToManeuverScale" params and recieve resultCode = "SUCCESS"
+--
+-- Expected:
+-- Params "distanceToManeuver" and "distanceToManeuverScale" of the mobile RPC "ShowConstantTBT" 
+-- should be mandatory 
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function ptuFunc(tbl)
+	tbl.policy_table.app_policies["0000001"].groups = {"Base-4", "Navigation-1"}
+end
+
+local function checkShowConstantTBT(pParams, pResultTable)
+	local cid = common.getMobileSession():SendRPC("ShowConstantTBT", pParams)
+	common.getMobileSession():ExpectResponse(cid, pResultTable)
+end
```

Comment:
@KrupenichOleg This steps are not enough for successful result. Please see full flow in https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase_SUCCESS.lua

Reply:
@dboltovskyi all done in commit [93b6aab](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2120/commits/93b6aab45df5f86daf2a7558e1d365a571158952)